src/api.js

```diff
@@ -377,20 +377,19 @@ api.declare({
   return res.reply({installed: false});
 });
 
-// Below is the draft code to make the badges clickable.
-// In the finished thing, the helper function will go to the beginning of the module,
-// and will be used in other APIs as well.
 api.declare({
-  name: 'taskLink',
-  title: 'Redirects to the task inspector page',
+  name: 'status',
+  title: 'Status Page for Branch',
   description: [
-    'Finds a link to the task inspector for the given task group',
-    'in the status object returned by GitHub,',
-    'and returns the link to redirect the user to that page.',
+    'For a given branch of a repository, this will always point',
+    'to a status page for the most recent task triggered by that',
+    'branch.',
+    '',
+    'Note: This may be a redirect rather than a direct link.',
   ].join('\n'),
   stability: 'experimental',
   method: 'get',
-  route: '/taskLink/:owner/:repo/:branch',
+  route: '/repository/:owner/:repo/:branch',
```

Comment:
All of the API methods start with `/repository` :)

But you're right - it seems like it should be `/repository/:owner/:repo/:branch/latest` or something like that, to parallel the `badge.svg` route.